src/components/common/state-machine/state-machine.ts

```diff
@@ -415,6 +415,7 @@ const authStateMachine = createMachine(
             PATH_NAMES.SECURITY_CODE_INVALID,
             PATH_NAMES.SECURITY_CODE_REQUEST_EXCEEDED,
             PATH_NAMES.CHECK_YOUR_EMAIL_CHANGE_SECURITY_CODES,
+            PATH_NAMES.MFA_RESET_WITH_IPV,
```

Comment:
Doe we need '_WITH_IPV'?

Reply:
I think that for the time being it makes a distinction between the new journeys and the old one where IPV wasn't required. Happy to change it, but I think it would be handy to be explicit in some way about reproving identity being part of the journey.